ToNote/Views/AddNoteDialogView.xaml

```diff
@@ -21,5 +21,6 @@
             </TextBox.InputBindings>
         </TextBox>
         <Button Command="{Binding AddNoteCommand}" CommandParameter="{Binding RelativeSource={RelativeSource Mode=FindAncestor, AncestorType=Window}}" Content="Add" Grid.Column="2"/>
+        <TextBlock Text="Note name cannot contain any of \ / : &lt; &gt; ? | characters" HorizontalAlignment="Left" VerticalAlignment="Center" FontSize="10" Foreground="Red" Margin="5,0,0,45" Grid.Column="1" Height="15" Width="225"/>
```

Comment:
This looks rather ugly. Also, it should only be visible when the user actually inputs a disallowed character, which could be accomplished simply by having a boolean in the Add Note viewmodel that got switched to true, if you try to add a note with an invalid name. This is why you should move the logic to the viewmodel itself, and execute it before calling CloseDialogWithResult.